srcpkgs/blanket/template

```diff
@@ -9,6 +9,6 @@ short_desc="Improve focus and productivity by listening to different sounds"
 maintainer="Reback00 <reback00@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/rafaelmardojai/blanket"
-distfiles="https://github.com/rafaelmardojai/blanket/archive/${version}.tar.gz"
-checksum=79942ed919507c0078266d221c433ba9209f8ea5f11371d55d64bd3b469623bf
+distfiles="https://github.com/rafaelmardojai/blanket/archive/refs/tags/${version}.tar.gz"
```

Comment:
why did you change distfiles location?

Reply:
the old one wouldn't work when I tested it

Reply:
it works for me, can you try again?

Reply:
I am so sorry it does work,. let me fix that right away.

Reply:
I have fixed it and tested it and it still works

Reply:
While it does work, I think it won't work, soon. As of it is now, GitHub links to new format in the webpage.

Reply:
So should it in fact be the old fomat?

Reply:
oh, I am sorry for my wrong review

Reply:
Ok i'll just switch it back, I also fixed the trailing whitespace